subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -572,4 +576,78 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
         expect:
         succeeds("mytest", "assertNoTestClasses")
     }
+
+    def "multiple getTestingFramework() calls on a test suite return same instance"() {
+        given:
+        buildFile << """
+            plugins {
+                id 'java'
+            }
+
+            def first = testing.suites.test.getTestingFramework()
```

Comment:
Your method rename did not propagate here, I believe that's the test failures you are seeing.

Reply:
That definitely broke this test, thanks.